tests/src/test/scala/whisk/core/containerpool/docker/test/DockerContainerTests.scala

```diff
@@ -363,11 +365,13 @@ class DockerContainerTests
     val id = ContainerId("id")
     val container = new DockerContainer(id, ContainerAddress("ip"), false)
 
-    container.suspend()
-    container.resume()
+    val suspend = container.suspend()
+    val resume = container.resume()
 
-    (docker.pause(_: ContainerId)(_: TransactionId)).verify(id, transid)
-    (docker.unpause(_: ContainerId)(_: TransactionId)).verify(id, transid)
+    Await.ready(for (_ <- suspend; _ <- resume) yield {
+      (docker.pause(_: ContainerId)(_: TransactionId)).verify(id, transid)
+      (docker.unpause(_: ContainerId)(_: TransactionId)).verify(id, transid)
+    }, 500.milliseconds)
```

Comment:
Have you verified this can actually fail? `Await.ready` only waits for the future to finish, but you're verifying in the `yield` bit, which desugars to a `map`.

Reply:
Oh, you are right. I totally missed that. 🤦‍♂️